<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2c3e50;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QComboBox" name="combo_ciclos"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white; font-weight: bold;</string>
         </property>
         <property name="text">
          <string>HORARIOS</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_profesores">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:left;
color:lightgrey;</string>
         </property>
         <property name="text">
          <string>Profesores</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_modulos">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:left;
color:lightgrey;</string>
         </property>
         <property name="text">
          <string>MÃ³dulos</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_horarios">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">text-align:left;
color:lightgrey;</string>
         </property>
         <property name="text">
          <string>Horarios</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>367</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="content_area" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: #f4f6f9;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page_profesores">
          <widget class="QTableWidget" name="tabla_profesores">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>431</width>
             <height>441</height>
            </rect>
           </property>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Ciclos</string>
            </property>
           </column>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>432</width>
             <height>25</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btn_agregar_profe">
              <property name="text">
               <string>Agregar Profesor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_editar_profe">
              <property name="text">
               <string>Editar Profesor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_borrar_profe">
              <property name="text">
               <string>Borrar Profesor</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="page_modulos">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>40</y>
             <width>171</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Modulos del ciclo seleccionado</string>
           </property>
          </widget>
          <widget class="QTableWidget" name="tabla_modulos">
           <property name="geometry">
            <rect>
             <x>-10</x>
             <y>90</y>
             <width>451</width>
             <height>441</height>
            </rect>
           </property>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Nombre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Horas Semanales</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Horas Diarias</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="page_horarios">
          <widget class="QTableWidget" name="tabl_horario_grid">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>80</y>
             <width>721</width>
             <height>521</height>
            </rect>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <row>
            <property name="text">
             <string>Nueva fila</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8:00</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>10:00</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>Lunes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>MArtes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Martes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Miercoles</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Viernes</string>
            </property>
           </column>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Horario de...</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>50</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>DAM1 / DAM2/...</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
